dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/config/SSLConfigManager.java

```diff
@@ -378,8 +379,18 @@ private SSLConfig parseDefaultSecureSocketLayer() throws Exception {
 
         // Obtain miscellaneous attributes from system properties
         String sslProtocol = getSystemProperty(Constants.SSLPROP_PROTOCOL);
-        if (sslProtocol != null && !sslProtocol.equals(""))
-            sslprops.setProperty(Constants.SSLPROP_PROTOCOL, sslProtocol);
+        if (sslProtocol != null && !sslProtocol.equals("")) {
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled())
+                Tr.debug(tc, "Setting default SSLProtocol: " + sslProtocol);
+
+            // Use PROTOCOL_TLSV1_2 if FIPS 140-3 is enabled
+            if (CryptoUtils.isFips140_3Enabled()) {
+                sslprops.setProperty(Constants.SSLPROP_PROTOCOL, Constants.PROTOCOL_TLSV1_2);
```

Comment:
Should this be `PROTOCOL_TLS_FIPS` instead of `PROTOCOL_TLS1_2`?

Reply:
Yes, good catch since 140-3 support TLSV1_2 and 1_3